homeassistant/components/isy994/helpers.py

```diff
@@ -416,7 +428,7 @@ def convert_isy_value_to_hass(
     if uom in (UOM_DOUBLE_TEMP, UOM_ISYV4_DEGREES):
         return round(float(value) / 2.0, 1)
     if precision not in ("0", 0):
-        return round(float(value) / 10 ** int(precision), int(precision))
+        return int(round(float(value) / 10 ** int(precision), int(precision)))
```

Comment:
Not sure what's the idea here, but if we only want an integer, round does return that if we don't pass any second parameter. But why not keep it as a float?

Reply:
We want a float. This is a refactoring error. Thanks for pointing it out 👍

Reply:
`homeassistant/components/isy994/helpers.py:431: error: Returning Any from function declared to return "Union[float, int, None]"  [no-any-return]`

It should have been `float` instead of `int`